examples/helm/hop/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
```

Comment:
$ helm lint .
==> Linting .
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, 0 chart(s) failed

Can we add hop logo as icon? @hansva @mattcasters  please give the image url.

Reply:
All Hop icons can be found here:  http://hop.apache.org/tech-manual/latest/hop-logo-and-icons.html

Reply:
Icon added. Is [this](https://hop.apache.org/tech-manual/latest/_images/logo/CMYK/jpg/HOP_logo_CMYK-4.jpg) the correct one?